### DISCUSSION TITLE
[Suggestion/Wish] Beans FrontPacks

### DISCUSSION THREAD

**Discussion by JasdanVM** [BeansGalaxy/Beans-Backpacks-2#181] [Feedback]
Yeah, quite a weird title, but I'm serious, could you add a config to enable using Beans Backpack on a special frontal Trinket?
As a Project Zomboid player is only natural for me to be greedy and put on as many containers as possible on me, while we don't usually see Fanny Packs in Minecraft, your bags are small enough, and pretty nice looking, so I would like to be able of wearing them on the front, while I have some other, more cumbersome Backpack... on the Back.
![2024-07-11_01 43 09](https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/109768767/9f479a58-b79f-4748-a98a-4f02762d44e5)
(Picture taken after I accidentally picked the Bean Backpack using "Carry On", not the inspiration for the idea, but it fits nicely to ilustrate it.)

**Comment by JasdanVM** [comment]
I don't mean to imply that I would rather use other backpacks than yours, quite the contrary, I like them so much that I wish I wasn't forced to make that choice, and similarly other players, and in large modpacks, could have all the cakes they'd like to eat.

**Comment by BeansGalaxy** [comment]
Hope you found a mod that suits what you needed, I was MIA for a while and just now seeing this. I love the Idea, I used to make a ton of "Ultra Hardcore" modpacks and in a IRL survival situation I'd be rocking as many backpacks as I could. 

I think I'll be making the announcement that the 1.20 version is officially dead soon, I've recently looked at the code again and I have no clue what any of it means so I think it's not worth saving. With that being said I will be adding Trinkets/Curios support very soon in the 1.21 version and I'll see how I could implement this. I've gotten a lot smarter with my code too and made it much more data driven so Mod Pack Creators can modify it as they need.

The 1.21 version is out in alpha for now, I wouldn't put it in any Mod Packs or even create a survival world with it since so much is still subject to change but if you want to check it out --> https://modrinth.com/mod/beans-backpacks-3

**Reply by JasdanVM** [reply]
Ah, damn, and here I am still stuck on 1.20.1
But it's nice that you liked the idea.
